app/views/shared/_batch.html.erb

```diff
@@ -1,99 +1,78 @@
 
-  <table class="sortable table">
-    <thead>
-      <tr>
-        <% if edit -%>
-          <th width="3%">Drag</th>
-        <% elsif stage %>
-          <th class="nosort" width="7%"><input type='checkbox' class='select_all'></th>
+<table  id="batch-<%= edit ? 'edit' : 'show' %>"
+        class="table"
+        data-batch-id="<%= @batch&.id %>">
```

Comment:
Removed the sortable class, and we'll mount Datatables by id instead. This allows us to mount the edit page with different options.